cmd/evm/main.go

```diff
@@ -227,15 +227,17 @@ type ruleSet struct{}
 
 func (ruleSet) IsHomestead(*big.Int) bool { return true }
 
-func (ruleSet) IsAtlantis(*big.Int) bool {
-	// Default true for tests
-	return true
-}
+//set IsAtlantis to true by default for tests
+func (ruleSet) IsAtlantis(*big.Int) bool { return true }
+
+//set IsAgharta to true by default for tests
+func (ruleSet) IsAgharta(*big.Int) bool { return true }
 
 func (ruleSet) GasTable(*big.Int) *vm.GasTable {
 	return &vm.GasTable{
 		ExtcodeSize:     big.NewInt(700),
 		ExtcodeCopy:     big.NewInt(700),
+		ExtcodeHash:     big.NewInt(400),
```

Comment:
Should add a new gas table for Agharta instead of just adding it on top of the Atlantis one. Although it won't be functionally different since Extcodehash isn't enabled previously, it would be logically incorrect? 

Similar to what you did here:
https://github.com/etclabscore/go-ethereum/pull/4/files#diff-8337e1399854c6a050cd98f558f9ce68R237